app/hotels/src/singleHotel/bookNow/BookNow.js

```diff
@@ -81,7 +82,9 @@ export class BookNow extends React.Component<Props> {
         <View style={styles.buttonWrapper}>
           <TouchableItem onPress={this.handleGoToPayment} activeOpacity={0.6}>
             <View style={styles.buttonInnerWrapper}>
-              <Text style={styles.buttonTitle}>Book Now</Text>
+              <Text style={styles.buttonTitle}>
+                {TransformButtonText('Book Now')}
```

Comment:
Isn't better to use components? Something like `<ButtonText />` sounds better to me than functions again.